.github/workflows/issue-triage.yml

```
@@ -81,4 +81,32 @@ jobs:
             [`repl`](https://babeljs.io/repl/), or if the `repl` is
             insufficient, a new and minimal repository with instructions on
             how to build/replicate the issue.
-        
+
+  npm_cache:
+    name: Npm Cache
+    runs-on: ubuntu-latest
+    steps:
+      - name: Npm Cache Comment
+        uses: babel/actions/create-comment@v2
+        if: |
+          github.event.action == 'labeled' &&
+          github.event.label.name == 'i: npm cache'
+        with:
+          token: ${{ secrets.BOT_TOKEN }}
+          issue: ${{ github.event.issue.number }}
+          comment: >
+            Hi @${{ github.event.issue.user.login }}!
+
+            This seems to be an npm related issue: you can verify that the package is available using the npm website.
+            For example, https://www.npmjs.com/package/@babel/types?activeTab=versions is the URL for `@babel/types`
+            (replace `@babel/types` in the URL with the package you want to check). If you see zero downloads don't worry:
+            npm can take up to a few days to display the downloads count, but this does not mean that it's not working.
+            
+            Although we can't handle it, we provide some common workarounds for your reference.
```

_Technically_ none of the following are really workarounds, howabout:

```suggestion
            Here are some things you can try:
```